src/main.js

```diff
@@ -44,9 +45,71 @@ function ymlContentToConfig(ymlContent) {
   }
 
   if (configJson.resource) {
-    config.source = configJson.resource;
+    config.source = removeTrailingSlashes(configJson.resource);
```

Comment:
We can't just remove slashes from a resource. There is semantic difference between `http://exampl.org/test` and `http://example.org/test/. In the context of Solid, the former is a resource and the latter is a container. See [here](https://solidproject.org/TR/protocol#resources) for more details.